### PR TITLE
Changed version numbers to reflect W2O fork

### PR DESCRIPTION
**Pull request by robdouglas**

### CODE REVIEW

File: facebook4j-core/pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.facebook4j</groupId>
   <artifactId>facebook4j-core</artifactId>
-  <version>2.1.1-SNAPSHOT</version>
```
Comment (sieverssj) [inline facebook4j-core/pom.xml:5]: This should be 2.1.1-W2O-SNAPSHOT to match the original version

**Comment by robdouglas** [comment]
Responded to code review feedback

File: pom.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.facebook4j</groupId>
   <artifactId>facebook4j</artifactId>
-  <version>2.0.5-SNAPSHOT</version>
+  <version>2.1.1-W2O-SNAPSHOT</version>
```
Comment (sieverssj) [inline pom.xml]: This one should stay 2.0.5-W2O-SNAPSHOT.  The module versioning in this project is not consistent, but I'd just honor it anyway.

**Comment by robdouglas** [comment]
Code Review feedback

**Comment by sieverssj** [comment]
:+1:

**Pull request merged**